CMakeLists.txt

```diff
@@ -18,10 +18,11 @@ add_custom_command(
 FIND_PACKAGE(PkgConfig REQUIRED)
 PKG_CHECK_MODULES(GTKMM3 gtkmm-3.0 REQUIRED)
 PKG_CHECK_MODULES(WEBKIT webkit2gtk-4.0 REQUIRED)
-include_directories(${GTKMM3_INCLUDE_DIRS} ${WEBKIT_INCLUDE_DIRS})
+PKG_CHECK_MODULES(APP_INDICATOR appindicator-0.1 REQUIRED)
```

Comment:
It worked on my side with appindicator3-0.1 pkg and libappindicator-gtk3 package installed. Could you check this please if this is the common package for all distros?

Reply:
We need to change the package to libappindicator3-dev for Ubuntu and GitHub. Then it will work correctly everywhere.